apps/betterangels-backend/clients/tests/test_permissions.py

```diff
@@ -314,6 +320,39 @@ def test_view_client_documents_permission(self, user_label: str, expected_docume
         else:
             self.assertTrue("errors" in response)
 
+    @parametrize(
+        "user_label, should_succeed",
+        [
+            ("org_1_case_manager_1", True),  # Owner should succeed
+            ("org_1_case_manager_2", True),  # Other CM in owner's org should succeed
+            ("org_2_case_manager_1", False),  # CM in different org should not succeed
+            ("client_user_1", False),  # Client modifying client profile should not succeed
```

Comment:
you can change this to
```suggestion
            ("non_case_manager_user", False),  # Client modifying client profile should not succeed
```
and undo the updates to `apps/betterangels-backend/common/tests/utils.py` below. We used to use `client_user_1` in these tests but we removed client users. Now we use `non_case_manager_user`. The [example in my branch](https://github.com/BetterAngelsLA/monorepo/compare/main...DEV-1674/update-document-filename#diff-511f398a3a6759dab6480621ca80e41db06205b2ceebe8aa864c66a6e65a8a76R242) was written before we made this change, which is why it's still in there